<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Projeto</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="separator"/>
    <addaction name="actionEditTIM"/>
    <addaction name="actionTIMDiagram"/>
    <addaction name="separator"/>
    <addaction name="actionManageFiles"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDocuments">
    <property name="title">
     <string>Documento</string>
    </property>
    <addaction name="actionNewDocument"/>
   </widget>
   <widget class="QMenu" name="menuClauses">
    <property name="title">
     <string>Cláusula</string>
    </property>
    <addaction name="actionNewClause"/>
    <addaction name="actionPathDiagram"/>
    <addaction name="actionTotalDiagram"/>
   </widget>
   <widget class="QMenu" name="menuConsolidation">
    <property name="title">
     <string>Consolidação</string>
    </property>
    <addaction name="actionProjectConsolidation"/>
    <addaction name="actionUpdateConsolidation"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuDocuments"/>
   <addaction name="menuClauses"/>
   <addaction name="menuConsolidation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>Novo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveProject_as">
   <property name="text">
    <string>Salvar como...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionManageFiles">
   <property name="text">
    <string>Arquivos importados</string>
   </property>
  </action>
  <action name="actionEditTIM">
   <property name="text">
    <string>Editar TIM</string>
   </property>
  </action>
  <action name="actionTIMDiagram">
   <property name="text">
    <string>Diagrama TIM</string>
   </property>
  </action>
  <action name="actionNewDocument">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Novo Documento</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNewClause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Nova Cláusula</string>
   </property>
  </action>
  <action name="actionCheckSpaces">
   <property name="text">
    <string>Lacunas</string>
   </property>
  </action>
  <action name="actionCheckGranularity">
   <property name="text">
    <string>Granularidade</string>
   </property>
  </action>
  <action name="actionSuspectClauses">
   <property name="text">
    <string>Cláusulas suspeitas</string>
   </property>
  </action>
  <action name="actionRedundantPaths">
   <property name="text">
    <string>Caminhos redundantes</string>
   </property>
  </action>
  <action name="actionPathDiagram">
   <property name="text">
    <string>Diagrama de caminho</string>
   </property>
  </action>
  <action name="actionTotalDiagram">
   <property name="text">
    <string>Diagrama total</string>
   </property>
  </action>
  <action name="actionProjectConsolidation">
   <property name="text">
    <string>Nova Consolidação de Projeto</string>
   </property>
  </action>
  <action name="actionIndexHistory">
   <property name="text">
    <string>HIstórico de Índices</string>
   </property>
  </action>
  <action name="actionUpdateConsolidation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Atualizar consolidação anterior</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
